dashboard/app/helpers/levels_helper.rb

```diff
@@ -320,7 +320,7 @@ def app_options
         {app: 'lab2', channel: view_options[:channel]}
       elsif @level.is_a? Blockly
         blockly_options
-      elsif @level.is_a?(Weblab) || @level.is_a?(Fish) || @level.is_a?(Ailab) || @level.is_a?(Javalab)
+      elsif @level.is_a?(Weblab) || @level.is_a?(Fish) || @level.is_a?(Ailab) || @level.is_a?(Javalab) || @level.is_a?(Aichat)
```

Comment:
Is this needed since this is a Lab2 app?

Reply:
I don't think so, thanks! I should also be able to remove the `non_blockly_puzzle_level_options` method from the `Aichat` model.